core/math/math_funcs.h

```diff
@@ -711,6 +711,22 @@ class Math {
 		}
 		return p_target;
 	}
+
+	// Divides num / dem and the result is rounded up:
+	// Examples:
+	//		0 / 64 = 0
+	//		63 / 64 = 1
+	//		64 / 64 = 1
+	//		65 / 64 = 2
+	//
+	// Will not work correctly if numerator + denominator - 1u results in an overflow.
+	static _ALWAYS_INLINE_ uint32_t divide_round_up(uint32_t numerator, uint32_t denominator) {
```

Comment:
```suggestion
	static _ALWAYS_INLINE_ uint32_t divide_round_up(uint32_t p_numerator, uint32_t p_denominator) {
```
Prefix arguments with `p_`, same everywhere

Reply:
Arg!

My bad.